docs/src/main/asciidoc/hibernate-orm-panache-kotlin.adoc

```diff
@@ -24,7 +24,7 @@ change or two.  To Panache-enable your entity, you would define it something lik
 [source,kotlin]
 ----
 @Entity
-class Person: PanacheEntity {
```

Comment:
Are u sure this is needed in kotlin?

Is it not possible to use a `data` class?

Reply:
This is just a fix of the documentation that misses the parenthesis.

Whether or not data classes can be used with Hibernate is a question for @evanchooly and I guess, as he is the author of the guide, that they are not a good fit for it.